test/unit/services/amazon_mws_test.rb

```diff
@@ -508,6 +508,12 @@ def test_retry_on_error
     refute_predicate response, :success?
   end
 
+  def test_read_timeout_longer_than_default
```

Comment:
Why approach this generically versus be explicit about the condition that is being tested for Amazon? I.e. `20`.

If the default changed to `25` tomorrow would we expect this test to fail? Would we want it to fail?